types/V1.mli

```diff
@@ -1,6 +1,6 @@
 (*
- * Copyright (c) 2011-2014 Anil Madhavapeddy <anil@recoil.org>
- * Copyright (c) 2013      Thomas Gazagnaire <thomas@gazagnaire.org>
+ * Copyright (c) 2011-2015 Anil Madhavapeddy <anil@recoil.org>
+ * Copyright (c) 2013-2015 Thomas Gazagnaire <thomas@gazagnaire.org>
```

Comment:
to be honest, I'm just doing this (as a bad way) to keep track of the active projects.